admin/templates/base.html

```diff
@@ -55,6 +55,9 @@
                 </a>
               </li>
               <li>
+                <a href="{% url 'password_change' %}" class="btn btn-flat">Change password</a>
```

Comment:
I know this is probably not a good thing to skip, but we should probably have a different base page and user page where admin users can update their information.

So for now a button on the base page will be okay, but we should add a ticket for user settings.